rest/changestest/changes_api_test.go

```diff
@@ -3856,6 +3856,62 @@ func TestCacheCompactDuringChangesWait(t *testing.T) {
 	longpollWg.Wait()
 }
 
+func TestTombstoneResync(t *testing.T) {
+	base.LongRunningTest(t)
+	t.Skip() // skip until CBG-3235 is resolved
+	if base.UnitTestUrlIsWalrus() {
+		t.Skip("Walrus does not support Xattrs")
+	}
+
+	if !base.TestUseXattrs() {
+		t.Skip("If running with no xattrs compact acts as a no-op")
+	}
+
+	var rt *rest.RestTester
+	if base.TestsUseNamedCollections() {
+		rt = rest.NewRestTesterMultipleCollections(t, &rest.RestTesterConfig{
+			DatabaseConfig: &rest.DatabaseConfig{DbConfig: rest.DbConfig{
+				QueryPaginationLimit: base.IntPtr(5),
+				Unsupported:          &db.UnsupportedOptions{UseQueryBasedResyncManager: true}}}}, 2)
+	} else {
+		rt = rest.NewRestTester(t, &rest.RestTesterConfig{
+			DatabaseConfig: &rest.DatabaseConfig{DbConfig: rest.DbConfig{
+				QueryPaginationLimit: base.IntPtr(5),
+				Unsupported:          &db.UnsupportedOptions{UseQueryBasedResyncManager: true}}},
+		})
+	}
```

Comment:
This collection-specific code isn't required as both configs are the same and you don't need to know the number of collections in use for assertions like the test you got this setup code from.

```suggestion
	rt := rest.NewRestTester(t, &rest.RestTesterConfig{
		DatabaseConfig: &rest.DatabaseConfig{DbConfig: rest.DbConfig{
			QueryPaginationLimit: base.IntPtr(5),
			Unsupported:          &db.UnsupportedOptions{UseQueryBasedResyncManager: true}}},
	})
```

Reply:
https://github.com/couchbase/sync_gateway/pull/6340/commits/b3111c121362d3ddd55aaa0acb9344f6c20178da